### ISSUE TITLE
[Feature] What is the meaning of  argument "multi_phased_distill_schedule" in distill.py

### ISSUE THREAD

**Issue by 1145284121** [hao-ai-lab/FastVideo#130]
### Motivation

What does the parameter `multi_phased_distill_schedule`in distill.py mean? Default is "4000-1", but 4000 seems to be unused; moreover, 1 corresponds to only 1 segment in PCM (I recall it being 4 or 8 in PCM). What is the reasoning behind this?

**Comment by 1145284121** [comment]
是PCM多分段的效果在视频上表现不好吗，或者为了支持step可调的推理？

**Comment by rlsu9** [comment]
Thanks for your feedback!

Yes, our ablation experiments suggest that multi-step distillation does not significantly improve quality. In this context, 4000 refers to the number of steps used for multi-phase distillation. For example, we first distill the model into 8 segments, then distill those into 4 segments. You can refer to [this experiment](https://github.com/hao-ai-lab/FastVideo/blob/experiments/scripts/experiment_7.sh#L50) for more details.

**Comment by 1145284121** [comment]
> Thanks for your feedback!
> 
> Yes, our ablation experiments suggest that multi-step distillation does not significantly improve quality. In this context, 4000 refers to the number of steps used for multi-phase distillation. For example, we first distill the model into 8 segments, then distill those into 4 segments. You can refer to [this experiment](https://github.com/hao-ai-lab/FastVideo/blob/experiments/scripts/experiment_7.sh#L50) for more details.

Thank you for your reply. So, 4000-1 is equivalent to directly distilling a consistency model with only one sub-segment using 4000 steps, right? Are the 6-step checkpoints of the released hunyuanvideo obtained through distillation from 4000-1?

**Comment by rlsu9** [comment]
1. 4000-1 is equivalent to directly distilling a consistency model with only one sub-segment using 4000 steps?
Yes
2. Are the 6-step checkpoints of the released hunyuanvideo obtained through distillation from 4000-1?
Yes, you can refer to [this script](https://github.com/hao-ai-lab/FastVideo/blob/dd75ee8509943f2872b1b196d245afe1961c629b/scripts/distill/distill_hunyuan.sh#L7) for our final distill recipe.

**Comment by 1145284121** [comment]
> 1. 4000-1 is equivalent to directly distilling a consistency model with only one sub-segment using 4000 steps?
>    Yes
> 2. Are the 6-step checkpoints of the released hunyuanvideo obtained through distillation from 4000-1?
>    Yes, you can refer to [this script](https://github.com/hao-ai-lab/FastVideo/blob/dd75ee8509943f2872b1b196d245afe1961c629b/scripts/distill/distill_hunyuan.sh#L7) for our final distill recipe.

Thank you very much for your response. I have another question: in  code, the 6-step reasoning does not iteratively add noise like LCM, but instead uses the Euler method for direct iterative solving. Is this because the Hunyuan pre-training is based on flow matching, and fine-tuning the v-prediction model yields better results? Or does your team have other considerations?

# compute the previous noisy sample x_t -> x_t-1
latents = self.scheduler.step(
    noise_pred, t, latents, **extra_step_kwargs, return_dict=False
)[0]

in step：
if self.config.solver == "euler":
        prev_sample = sample + model_output.to(torch.float32) * dt

**Comment by rlsu9** [comment]
Hi, I am not quite sure about ` iteratively add noise like LCM`. But I guess your question about denoise part might related to the section 4.1: Consistency Distillation from the [LCM paper](https://arxiv.org/pdf/2310.04378).

**Comment by 1145284121** [comment]
> add noise like LCM

thanks for your reply, `add noise like LCM`  refers to prediction of the zero point at each step during multi-step inference ,and add noise by $\hat{\mathbf{x}}_{r_{n}}\stackrel{\cdot}{\longleftrightarrow}\mathbf{x}+\sqrt{\tau_{n}^{2}-\epsilon^{2}}\mathbf{z}$ (as shown in the figure, which from the paper [Consistency Models](https://arxiv.org/abs/2303.01469)). In FastVideo, multi-step inference is achieved through the integration of v-prediction, i.e., v*dt, which appears to differ from the theory of Consistency Models?
![image](https://github.com/user-attachments/assets/d4866efa-0681-4b9b-976a-180003d39823)

**Comment by Rookienovice** [comment]
> > add noise like LCM
> 
> thanks for your reply, `add noise like LCM` refers to prediction of the zero point at each step during multi-step inference ,and add noise by $\hat{\mathbf{x}}_{r_{n}}\stackrel{\cdot}{\longleftrightarrow}\mathbf{x}+\sqrt{\tau_{n}^{2}-\epsilon^{2}}\mathbf{z}$ (as shown in the figure, which from the paper [Consistency Models](https://arxiv.org/abs/2303.01469)). In FastVideo, multi-step inference is achieved through the integration of v-prediction, i.e., v*dt, which appears to differ from the theory of Consistency Models? ![image](https://github.com/user-attachments/assets/d4866efa-0681-4b9b-976a-180003d39823)

Hi, Do you have any conclusions about the problem?

**Comment by SolitaryThinker** [comment]
For now we have deprecated our PCM recipe and have no current plans to support it in v1
